manifests/profile/copymod.pp

```diff
@@ -1,4 +1,4 @@
-define bootstrap::copymod(
```

Comment:
defined types don't fit into roles & profiles. This should probably remain `bootstrap::copymod`